resources/commitee-documentation/membership-committee-charter.md

```diff
@@ -0,0 +1,96 @@
+
+<p align="center">
+  <img src="../../img/membership-request.png" />
+</p>
+
+- On seeing this [Jessica White](https://github.com/wordshaker) will accept the issue and close it, then send you the next meetings invite.
+- After attending two meetings, you will be added to the CODEOWNERS of this repository and be added to the committee GitHub Teams.
+
+### Expected Level of Contribution
+
+To remain a member of the committee we ask that you attempt to attend meetings at least once a month. If you are unable to attend at this frequency or have to miss a couple for extenuating circumstances, you will be expected to raise this with the committee. You won't need to provide specifics, it's to let the group know you intend to take part in future.
+
+If you cannot attend at that frequency, you could also contribute by submitting issues or pull requests to the repository.
+
+Each committee meeting will have a draft agenda in GitHub available before the meeting. This is how to check when issues you are passionate about are going to be up for discussion and action votes.
+
+## Termination of Membership
+
+On a quarterly basis, the chair, scribe or appointed representative will review levels of contribution from [members of the membership GitHub Teams](https://github.com/orgs/dotnet-foundation/teams/membership/members). If the expected contribution levels have not been met, with no reasoning, your membership to this team will be revoked.
+
+After termination, you can re-apply to join the committee if you so wish.
+
+## Chairperson
+
+The Chairperson of the committee is an elected member of the Board of Directors. They will have volunteered to be the chair and spokesperson of the committee.
+
+The current Chairperson of the membership committee is [Jessica White](mailto:Jessica.White@dotnetfoundation.org)
+
+
+## Standard Committee Procedures
+
+The committee meets on the 1st and 3rd Wednesday of each month at 7:00pm UTC. The meetings are faciliated by the chairperson or a volunteer if the chairperson is not available to attend.
+
+Meetings have an agenda which is shared on GitHub and can be contributed to by any committee member. Each agenda item will have a GitHub Issue assigned which provides more context to the discussion point. 
+
+Minutes of each meeting are recorded, and can be viewed [here](../../meetings/).
+
+### Decisions and disputes
+Decisions are made via vote. If the committee cannot come to agreement on an action, it will be postponed or the chairperson will discuss with other board members to determine which decision would be in the best interests of the foundation.
+
+### Feedback to the board
+The Chairperson will feedback to the other members of the Board of Directors on a monthly basis. This includes topics that are currently in discussion, actions being taken and measures of success.
+
+## Duties, and Responsibilities
+
+The committee meets on the 1st and 3rd Wednesday of each month at 7:00pm UTC. 

```

Comment:
Is it worth revisiting the time periodically (maybe once a year?) That might make as committee members other commitments change, or as the makeup changes and different times work better in different time zones.

Reply:
https://github.com/dotnet-foundation/wg-membership/pull/89/commits/78dd4cc5ff059750bf826be49d21d12dc7438540